src/pages/index.js

```diff
@@ -103,101 +113,90 @@ const renCard = (data) => {
         );
         deleteCardPopup.open();
       },
+      handleLike: (card, isLiked) => {
+        cardElement.toggleLikeElement();
```

Comment:
`        cardElement.toggleLikeElement();
`

You need to **toggle likes** only in block `then` after a successful response from the server because if there is an error the user will not know about that and will see the toggled heart color.